### ISSUE TITLE
[BITV] 9.4.1.2/5.5 - The "Copy to clipboard" button should be implemented as a button and not as a link without an href attribute. An accessible name could then be provided via the aria-label attribute. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/activity#1111]
- [x] https://github.com/nextcloud/activity/pull/1123

![Image](https://user-images.githubusercontent.com/1315170/223739112-17af8ce7-1190-42ff-a1e5-54198119bdf3.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v5-n5

</details>

**Comment by AndyScherzinger** [comment]
Acceppted as no fix, see https://github.com/nextcloud/server/issues/35505#issuecomment-1331743638
